app/controllers/sessions_controller.rb

```diff
@@ -8,22 +8,22 @@ def new
 	  def create
 	  	user = User.find_by(username: params[:user][:username])
 	  	password = params[:user][:password]
-	
-	  	if user && (user.authenticate("#{password}") != false)
+
+	  	if user && user.authenticate("#{password}")
```

Comment:
if this works then user.authenticate(password) should also work. eitherway.

Reply:
Just a small simplification.